src/reporters/base.js

```diff
@@ -110,7 +114,7 @@ export default class BaseReporter {
         task.on('test-run-done', (testRun) => {
             var reportItem      = this._getReportItemForTestRun(testRun);
             var userAgent       = testRun.browserConnection.userAgent;
-            var testRunErrMsgs  = testRun.errs.map(err => this.formatter(err, userAgent));
+            var testRunErrMsgs  = testRun.errs.map(err => this.errorDecorator(err, userAgent));
```

Comment:
Shouldn't it be
```js
var testRunErrMsgs  = testRun.errs.map(err => format(err, userAgent, this.errorFormatter));
```
?
Oh, and currently `format` method should add information about browser in which test failed. So, we need to add this logic as well. And you need to fix `test/server/reporters-test.js` so it will use new approach (I think we will need to use error-like objects, instead of plain string which are used by tests right now).